src/scripts/Controller.js

```diff
@@ -586,6 +589,20 @@ export default class SitemapController {
 		return true;
 	}
 
+	copyText() {
+		var copyText = document.getElementById('copyText');
+		var range = document.createRange();
+		range.selectNode(copyText);
+		window.getSelection().removeAllRanges();
+		window.getSelection().addRange(range);
+		try {
+			document.execCommand('copy');
+		} catch (err) {
+			console.log('Can`t copy, boss');
+		}
+		document.execCommand('copy');
```

Comment:
You are doing `document.execCommand('copy')` twice, remove one.
I believe wrapping it in try-catch is not that necessary and we could let it raise.